src/models/shared-enums.ts

```diff
@@ -0,0 +1,36 @@
+export type VehicleType = "psv" | "hgv" | "trl" | "car" | "lgv" | "motorcycle";
+
+export type VehicleConfiguration =
```

Comment:
I need @m-ujaffer second pair of eyes on the definition of models please.
They also could be part of separate shared repo

Reply:
As agreed on chat with Jon, leaving this for a future PR and resolving for now - @SanjeetBJSS accepts the tech debt